Core/KSPManager.cs

```diff
@@ -151,6 +152,103 @@ public KSP AddInstance(KSP ksp_instance)
             return ksp_instance;
         }
 
+        /// <summary>
+        /// Clones an existing KSP installation.
+        /// </summary>
+        /// <param name="existing_instance">The KSP instance to clone.</param>
+        /// <param name="new_name">The name for the new instance.</param>
+        /// <param name="new_path">The path where the new instance should be located.</param>
+        public void CloneInstance (KSP existing_instance, string new_name, string new_path)
+        {
+            if (!existing_instance.Valid)
+            {
+                throw new NotKSPDirKraken(existing_instance.GameDir(), "The specified instance is not a valid KSP instance.");
+            }
+
+            try
+            {
+                log.Debug("Copying directory.");
+                Utilities.CopyDirectory(existing_instance.GameDir(), new_path, true);
+
+                // Add the new instance to the registry
+                KSP new_instance = new KSP(new_path, new_name, User);
+                AddInstance(new_instance);
+            }
+            // Thrown by AddInstance() if created instance is not valid.
+            // Thrown f.e. if something went wrong with copying.
+            catch (NotKSPDirKraken kraken)
+            {
+                throw kraken;
+            }
+            // Thrown by CopyDirectory() if directory doesn't exist. Shouldn't be thrown anytime.
+            catch (DirectoryNotFoundKraken kraken)
+            {
+                throw kraken;
+            }
+            // Thrown by CopyDirectory() if the specified folder already exists and is not empty.
+            catch (IOException e)
+            {
+                throw e;
+            }
+        }
+
+        /// <summary>
+        /// Create a new fake KSP instance
+        /// </summary>
+        /// <param name="new_name">The name for the new instance.</param>
+        /// <param name="new_path">The loaction of the new instance.</param>
+        /// <param name="version">The version of the new instance. Should have a build number.</param>
+        /// <param name="DLC">Whether to fake the DLC too.</param>
```

Comment:
Do we need an independent `DLC` parameter, or could we simply check whether `dlcVersion` is null? It doesn't look like passing these parameters in unexpected ways would be useful, e.g., DLC=true and dlcVersion=null, or DLC=false and dlcVersion="1.0.0".